compare/go-jira.py

```diff
@@ -86,17 +86,40 @@
 cloudera_jira = JIRA('https://issues.cloudera.org')
 
 IGNORABLE_KEYS = {'16x16', '24x24', '32x32', '48x48', 'timeZone', 'displayName'}
-IGNORABLE_PATHS = {'priority.iconUrl', 'comment.comments.self', 'comment.comments.id',
-                   'resolution.description', 'fixVersions.self', 'fixVersions.id',
-                   'versions.self', 'versions.id', 'issuetype.iconUrl',
-                   'reporter.emailAddress', 'project.id', 'assignee.emailAddress',
-                   'comment.comments.updateAuthor.emailAddress',
-                   'comment.comments.author.emailAddress', 'creator',
-                   'issuetype.avatarId', 'project.self', 'status.description',
-                   'resolution.id', 'resolution.self', 'components.self', 'components.id',
-                   'comment.comments.updated', 'issuetype.self', 'issuetype.id',
+IGNORABLE_PATHS = {'assignee.emailAddress',
                    'attachment', # attachments are checked separately
-}
+                   'comment.comments.author.emailAddress',
+                   'comment.comments.id',
+                   'comment.comments.self',
+                   'comment.comments.updateAuthor.emailAddress',
+                   'comment.comments.updated',
+                   'components.id',
+                   'components.self',
+                   'creator',
+                   'fixVersions.id',
+                   'fixVersions.self',
+                   'issuelinks.inwardIssue.fields.issuetype.iconUrl',
```

Comment:
The IMPALA cases are no longer in issues-test.apache.org, so I can't test that now.